submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/model/DbFile.kt

```diff
@@ -31,6 +31,8 @@ class DbFile(
 
     var size: Long = 0L
 
+    var directory: Boolean = false
```

Comment:
You should update the database schema (Schema.sql) with this change. Also, shouldn't we make these changes for the ReferencedFile object, which is used by the files lists?

Reply:
Actually this was supported so field is already there. I will create a ticket to add support in file list

Reply:
It's in the database but it's not in the Schema.sql file

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/infrastructure/src/main/resources/setup/database/Schema.sql#L28